modules/modules.nix

```
@@ -93,6 +93,7 @@ let
     (loadModule ./programs/lf.nix { })
     (loadModule ./programs/lsd.nix { })
     (loadModule ./programs/man.nix { })
+    (loadModule ./programs/mangohud.nix { })
```

Seems mangohud is Linux specific so should make the module conditional
```suggestion
    (loadModule ./programs/mangohud.nix { condition = hostPlatform.isLinux; })
```